webpack.config.js

```diff
@@ -3,17 +3,23 @@ const path = require('path');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-const WatchExternalFilesPlugin = require('webpack-watch-files-plugin').default;
+const Watchpack = require('watchpack');
 
 // get tuture path
 const tuturePath = process.env.TUTURE_PATH;
```

Comment:
It seems we didn't need to pass `TUTURE_PATH` around any more, so how about delete this out?